Rainier-2U-MRW.xml

```diff
@@ -130149,18 +130404,18 @@
 	</attribute>
 </targetInstance>
 <targetInstance>
-	<id>ddr4</id>
-	<type>unit-ddr4-jedec</type>
+	<id>ddrA</id>
+	<type>unit-ddr-jedec</type>
```

Comment:
My current code only supports "unit-ddr" not "unit-ddr-jedec".  I remember doing that because it isn't really a jedec anything inside the DDIMM so the name made no sense.  I can add the support (which has to go through before you can merge) or you can change.